cli/root.go

```diff
@@ -435,6 +436,13 @@ func (r *RootCmd) Command(subcommands []*serpent.Command) (*serpent.Command, err
 			Value:       serpent.BoolOf(&r.disableDirect),
 			Group:       globalGroup,
 		},
+		{
+			Flag:        varDisableNetworkTelemetry,
+			Env:         "CODER_DISABLE_NETWORK_TELEMETRY",
+			Description: "Disable network telemetry.",
+			Value:       serpent.BoolOf(&r.noNetworkTelemetry),
+			Group:       globalGroup,
```

Comment:
Since the flag isn't hidden you should put a better description here:
```
Disable network telemetry. Network telemetry is collected when connecting to workspaces in the CLI, and is forwarded to the server, which may forward it to Coder if telemetry is enabled on the server. Network telemetry is used to measure network quality and detect regressions.
```